sys-kernel/dracut/dracut-9999.ebuild

```diff
@@ -78,25 +79,21 @@ src_configure() {
 src_install() {
 	default
 
-	local dracutlibdir="usr/lib/dracut"
-
-	local libdirs="/$(get_libdir) /usr/$(get_libdir)"
+	local libdirs=( /$(get_libdir) /usr/$(get_libdir) )
 	if [[ ${SYMLINK_LIB} = yes ]]; then
 		# Preserve lib -> lib64 symlinks in initramfs
-		[[ $libdirs =~ /lib\  ]] || libdirs+=" /lib /usr/lib"
+		[[ ${libdirs[*]} == */lib" "* ]] || libdirs+=( /lib /usr/lib )
 	fi
 
-	einfo "Setting libdirs to \"${libdirs}\" ..."
-	echo "libdirs=\"${libdirs}\"" > "${T}/gentoo.conf"
-	insinto "${dracutlibdir}/dracut.conf.d"
+	einfo "Setting libdirs to \"${libdirs[*]}\" ..."
+	echo "libdirs=\"${libdirs[*]}\"" > "${T}/gentoo.conf" || die
+	insinto "usr/lib/dracut/dracut.conf.d"
 	doins "${T}/gentoo.conf"
 
 	insinto /etc/logrotate.d
 	newins dracut.logrotate dracut
 
-	dodir /var/lib/dracut/overlay
-
-	dodoc dracut.html
+	keepdir /var/lib/dracut/overlay
```

Comment:
It looks like this directory is used by the dracut-catimages command. Given that this is probably a rarely used command, and seems quite buggy, it might be better to just drop this line.